q2_cutadapt/_demux.py

```diff
@@ -57,7 +60,9 @@ def _build_demux_command(seqs_dir_fmt, barcode_fhs, per_sample_dir_fmt,
             # Dual indices
             cmd += [
                 '--pair-adapters',
-                '-G', 'file:%s' % barcode_fhs['rev'].name,
+                '-G',
+                # pylint: disable-next=line-too-long  # Increase readability
```

Comment:
Since we don't use pylint in our workflows and for the sake of simplicity could you remove this line?